hack/build-operator-podman.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+if [ -z "$ARCH" ] || [ -z "$PLATFORMS" ] || [ -z "$OPERATOR_IMAGE_TAGGED" ]; then
+    echo "Error: ARCH, PLATFORMS, and OPERATOR_IMAGE_TAGGED must be set."
+    exit 1
+fi
+
+IFS=',' read -r -a PLATFORM_LIST <<< "$PLATFORMS"
+
+# Remove any existing manifest and image
+podman manifest rm "${OPERATOR_IMAGE_TAGGED}" || true
+podman rmi "${OPERATOR_IMAGE_TAGGED}" || true
```

Comment:
please consider adding redir of stderr to /dev/null so it will clean up the logs from marking error on CI

Reply:
Added in PR: https://github.com/kubevirt/cluster-network-addons-operator/pull/1928